mod/update_contact.php

```diff
@@ -28,11 +28,10 @@
 
 function update_contact_content(App $a)
 {
-	if (!empty($_GET['force']) || !DI::pConfig()->get(local_user(), 'system', 'no_auto_update')) {
-		$text = Contact::content([], true);
+	if (!empty($a->argv[1]) && (!empty($_GET['force']) || !DI::pConfig()->get(local_user(), 'system', 'no_auto_update'))) {
+		$text = Contact::getConversationsHMTL($a, $a->argv[1], true, ($_GET['item'] ?? 0));
```

Comment:
Please make sure `$a->argv[1]` is set with a `??` for example.

Reply:
There is the `empty($a->argv[1])` for.

Reply:
Never mind.